nasa-gibs/worldview
# WV-3101: Add 2 PACE Orbit Track layers

## Description

Fixes #WV-3101

Add Ascending and Descending PACE Orbit Track layers.

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-187.36995530002213,-66.4859388048204,85.26885945737911,59.55598445737084&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,OrbitTracks_PACE_Descending,OrbitTracks_PACE_Ascending,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-04-23-T19%3A06%3A37Z)
2. Check to make sure the layers load and that you can take an image snapshot.

@nasa-gibs/worldview
